src/Roave/SecurityAdvisories/VersionConstraint.php

```diff
@@ -41,7 +40,7 @@ private function __construct()
      */
     public static function fromString(string $versionConstraint) : self
     {
-        $constraintString = (string) $versionConstraint;
+        $constraintString = strtolower((string) $versionConstraint);
```

Comment:
Buggy from before, but `$versionConstraint` is already a `string`, so another cast isn't needed here